apps/docs-snippets/src/guide/cookbook/signing-transactions.test.ts

```diff
@@ -0,0 +1,122 @@
+import type { Provider, BN } from 'fuels';
+import {
+  WalletUnlocked,
+  Predicate,
+  BaseAssetId,
+  Script,
+  FunctionInvocationResult,
+  ScriptTransactionRequest,
+} from 'fuels';
+
+import {
+  DocSnippetProjectsEnum,
+  getDocsSnippetsForcProject,
+} from '../../../test/fixtures/forc-projects';
+import { getTestWallet } from '../../utils';
+
+/**
+ * @group node
+ */
+describe('Signing transactions', () => {
+  let sender: WalletUnlocked;
+  let receiver: WalletUnlocked;
+  let signer: WalletUnlocked;
+  let provider: Provider;
+  let gasPrice: BN;
+  const { abiContents: abiPredicate, binHexlified: binPredicate } = getDocsSnippetsForcProject(
+    DocSnippetProjectsEnum.PREDICATE_SIGNING
+  );
+  const { abiContents: abiScript, binHexlified: binScript } = getDocsSnippetsForcProject(
+    DocSnippetProjectsEnum.SCRIPT_SIGNING
+  );
+
+  beforeAll(async () => {
+    sender = await getTestWallet();
+    signer = WalletUnlocked.generate({
+      provider: sender.provider,
+    });
+
+    provider = sender.provider;
+    ({ minGasPrice: gasPrice } = provider.getGasConfig());
+  });
+
+  beforeEach(() => {
+    receiver = WalletUnlocked.generate({
+      provider: sender.provider,
+    });
+  });
+
+  it('creates a transfer with external signer [script]', async () => {
+    const amountToReceiver = 100;
+
+    // #region multiple-signers-2
+    // #import { Script, BaseAssetId };
+
+    // Create invocation scope
+    const script = new Script(binScript, abiScript, sender);
+    const scope = script.functions
+      .main(signer.address.toB256())
+      .txParams({ gasPrice, gasLimit: 10_000 });
+    await scope.fundWithRequiredCoins();
+
+    // Create the transaction request
+    const transactionRequest = await scope.getTransactionRequest();
+    transactionRequest.addCoinOutput(receiver.address, amountToReceiver, BaseAssetId);
+
+    // Sign the transaction
+    await transactionRequest.addSigner(signer);
+
+    // Send the transaction
+    const response = await sender.sendTransaction(transactionRequest);
+    await response.waitForResult();
+    // #endregion multiple-signers-2
+
+    const { value } = await FunctionInvocationResult.build<BN>([scope], response, false, script);
+
+    expect(value.toNumber()).toEqual(1);
+    expect((await receiver.getBalance()).toNumber()).toEqual(amountToReceiver);
+  });
+
+  it('creates a transfer with external signer [predicate]', async () => {
+    const amountToReceiver = 100;
+
+    // #region multiple-signers-4
+    // Create and fund predicate
+    // #import { Predicate, BaseAssetId };
```

Comment:
Should we remove the "Create and fund predicate" comment?

It's appearing in front of the import, which is weird.

![line-break](https://github.com/FuelLabs/fuels-ts/assets/26660/101247d7-e7ca-4cb9-aca4-bfcdad9398e1)

@Torres-ssf Is this usage of having comments in-between context/import supported?

Reply:
Yes that should not be there, have removed.